core/src/main/scripts/importer/allowed_data_types.txt

```diff
@@ -32,3 +32,5 @@ METHYLATION	CONTINUOUS	methylation_hm450
 FUSION	FUSION	fusion
 PROTEIN_LEVEL	LOG2-VALUE	rppa
 PROTEIN_LEVEL	Z-SCORE	rppa_Zscores
+PROTEIN_LEVEL	CONTINUOUS	protein_quantification
+PROTEIN_LEVEL	Z-SCORE	protein_quantification_zscores
```

Comment:
should we keep the pattern and make this `protein_quantification_Zscores` with capital Z?